.changeset/calm-pears-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+'@stacks/wallet-web': patch
+---
+
+Center onboarding elements
```

Comment:
Please take a look at https://github.com/blockstack/stacks-wallet-web/blob/main/CHANGELOG.md, ideally these messages will be a bit more verbose than what would typically be found in a commit message :) we want to make reading the changelog fun and informative!

Reply:
Got it, thanks. I wasn't quite sure what I was supposed to include.